src/styles/blocks/card.scss

```
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $white-color;
+
+  &__screen {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    width: 166px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 36px 0 4px;
```

move margin 36px  to a n image as margin-bottom
```suggestion
    margin-bottom: 4px;
```